gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,5 @@
-#Mon Dec 09 09:39:29 GMT 2019
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.3-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

all is normally better, otherwise intellij complains it doesn't have sources
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```

you can add this to your build.gradle so that its automatic:

```
wrapper {
    distributionType = Wrapper.DistributionType.ALL
}
```